docs/github-app.md

```diff
@@ -0,0 +1,40 @@
+# Setting up the GitHub App
+
+The GitHub App is used by the release workflow to perform actions on your repos, like creating releases and
+making PR comments.
+
+Each organisation that uses the release workflow will need to create their _own_ GitHub App.
+If `gha-scala-library-release-workflow` had its own server infrastructure, we could probably follow the more
+common model of a single GitHub App being used by many organisations, but instead we take advantage of all those
+free GitHub Actions minutes, so we need to pass the workflow the private key of the GitHub App so that it can
+authenticate as the GitHub App... therefore we must each have our own GitHub App, so that we don't share private keys.
+
+## 1. Create the GitHub App
+
+### GitHub App for a single user account
+
+You can just click this link to get taken to a pre-filled page to create a new GitHub App - you'll just need to
+customise the app name:
+
+https://github.com/settings/apps/new?name=scala-library-release&url=https://github.com/guardian/gha-scala-library-release-workflow&public=false&contents=write&pull-requests=write&webhook_active=false
```

Comment:
Whoops, this should have been `pull_requests=write` (with an underscore), not `pull-requests=write` - fixed in https://github.com/guardian/gha-scala-library-release-workflow/pull/41